orttraining/tools/amdgpu/script/rpl_rc.xml

```diff
@@ -7,5 +7,4 @@ obj-tracking=on
 ></defaults>
```

Comment:
Is this file used anywhere, or can it be deleted?

Reply:
It was used for rocprof in docker container, e.g. rocprof -i rpl_rc.xml